app/views/admin/licenses/_form.html.erb

```diff
@@ -31,11 +31,7 @@
     <%= f.label :copyright_notice %><br>
     <%= f.text_area :copyright_notice %>
   </div>
-  <div class="field">
-    <%= f.label :is_public_domain %><br>
-    <%= f.check_box :is_public_domain %>
-  </div>
```

Comment:
Why? There is public domain content out there …

Reply:
I think public domain stuff would be implemented as `*_contract: null, *_notice: null, requires_*: false, allows_*: true`

The `is_public_domain` column apparently never existed.

Reply:
:+1: